python/lsst/ip/isr/ptcDataset.py

```diff
@@ -105,6 +106,9 @@ class PhotonTransferCurveDataset(IsrCalib):
     gainErr : `dict`, [`str`, `float`]
         Dictionary keyed by amp names containing the errors on the
         fitted gains.
+    gainList : `dict`, [`str`, `np.ndarray`]
```

Comment:
There is no `gainUnadjusted` added to this doc string. Also, may want to adjust the `gain`description to mention that it is now the amp-offset "adjusted" values, and specify that the `gainUnadjusted` values are the fitted values.